src/markdownwidget/plugin.ts

```diff
@@ -13,6 +13,8 @@ import {
   MarkdownWidgetFactory
 } from './widget';
 
+import './theme.css';
```

Comment:
Yep, always :)

On Sat, Jul 16, 2016, 16:29 Ryan Grout <notifications@github.com> wrote:

> In src/markdownwidget/plugin.ts
> <https://github.com/jupyter/jupyterlab/pull/427#discussion_r71072149>:
>
> > @@ -13,6 +13,8 @@ import {
> >    MarkdownWidgetFactory
> >  } from './widget';
> >
> > +import './theme.css';
>
> Thanks @jasongrout <https://github.com/jasongrout>. Are those imports
> supposed to be alphabetical?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/jupyter/jupyterlab/pull/427/files/3a573234a03368144d8220de2a156585ba5a1388#r71072149>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALwZnX7tTMg0qJoxcz_Timy5lm6nsViks5qWU09gaJpZM4JOGFq>
> .
>